@here/harp-mapview/lib/MapTileCuller.ts

```diff
@@ -88,15 +88,12 @@ export class MapTileCuller {
      */
     private getFrustumCorners(): THREE.Vector3[] {
         const frustumCorners = this.m_frustumCorners;
-
-        const camera = new THREE.Camera();
-
-        camera.projectionMatrix.copy(this.m_camera.projectionMatrix);
+        const invProjMatrix = (this.m_camera as any).projectionMatrixInverse as THREE.Matrix4;
 
         let cornerIndex = 0;
 
         function addPoint(x: number, y: number, z: number) {
-            frustumCorners[cornerIndex++].set(x, y, z).unproject(camera);
+            frustumCorners[cornerIndex++].set(x, y, z).applyMatrix4(invProjMatrix);
```

Comment:
You can use `.unproject(this.m_camera)` here, as it takes camera's projectionMatrixInverse now.
See https://github.com/mrdoob/three.js/blob/dev/src/math/Vector3.js#L320,
and PR https://github.com/mrdoob/three.js/pull/15996/files merged in r103

Reply:
unproject is also multiplying with the world matrix. This we don't want to do in this case, that's why in the old code we created a temporary camera.

Reply:
Yep, I've missed that. Maybe some comment for clarity?